develop/docs/dlq.md

```diff
@@ -0,0 +1,28 @@
+# Managing DLQ Using tdbg
+When a task in the Temporal server encounters a terminal error, it may be enqueued in the Dead Letter Queue (DLQ). 
+To effectively manage these DLQ messages, follow these steps:
+
+1. Search `Task enqueued to DLQ` in logs and see task details.
```

Comment:
Also mention that they will know something has been DLQ'd by the metric being incremented and then point to the metric